net/s3cmd/Portfile

```
@@ -22,6 +22,7 @@ long_description    S3cmd lets you copy files from/to Amazon S3 \
                     of Amazon's CloudFront content delivery network.
 
 homepage            http://s3tools.org/s3cmd
```

Can we make this HTTPS while making this change?

```suggestion
homepage            https://s3tools.org/s3cmd
```

---

Good catch, thanks! Sure.